receiver/kubeletstatsreceiver/metadata.yaml

```diff
@@ -187,6 +187,48 @@ metrics:
       monotonic: true
       aggregation_temporality: cumulative
     attributes: []
+  k8s.node.runtime.imagefs.available:
+    enabled: false
+    description: "Node runtime imagefs available"
+    unit: By
+    gauge:
+      value_type: int
+    attributes: []
+  k8s.node.runtime.imagefs.capacity:
+    enabled: false
+    description: "Node runtime imagefs capacity"
+    unit: By
+    gauge:
+      value_type: int
+    attributes: []
+  k8s.node.runtime.imagefs.used:
+    enabled: false
+    description: "Node runtime imagefs used.This may not equal CapacityBytes - AvailableBytes because there are certain system-level operations or processes that may reserve a portion of the available space for their own use"
```

Comment:
```suggestion
    description: "Node runtime imagefs used. This may not equal CapacityBytes - AvailableBytes because there are certain system-level operations or processes that may reserve a portion of the available space for their own use"
```